### DISCUSSION TITLE
Lag spikes in menus

### DISCUSSION THREAD

**Discussion by Non-Binary-Programmer** [ppy/osu#24635] [Q&A] [answered]
I experienced two lag spikes in quick succession: one in the mod select menu, then another in the settings menu. The one in the settings menu was longer (several seconds long).

[updater.log](https://github.com/ppy/osu/files/12424110/updater.log)
[database.log](https://github.com/ppy/osu/files/12424112/database.log)
[input.log](https://github.com/ppy/osu/files/12424114/input.log)
[network.log](https://github.com/ppy/osu/files/12424116/network.log)
[performance.log](https://github.com/ppy/osu/files/12424118/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12424120/runtime.log)

**Comment by peppy** [comment, marked as answer]
Performance and optimisation considerations are an ongoing effort.
